<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="ShowLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>711</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="SortLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>671</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the name and the mark
 of student:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="assessmenLine">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="SortTimeLabel">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
